udata/forms/fields.py

```diff
@@ -115,7 +127,13 @@ class FileField(FieldHelper, fields.FileField):
 
 
 class URLField(FieldHelper, EmptyNone, html5.URLField):
-    pass
+    def pre_validate(self, form):
```

Comment:
Yes, I am sure. I implemented this behavior because the error was not detected at the form level, but at the database level, which raised a 500.

It is easy to use wtforms.validators.URL but I am not in favor of it:
- The validation is much lighter than at the Mongo level. For foobarbaz fails, but htt://www.lemonde.fr passes. If the data submitted by the client is not invalidated by the form, the client receives a 500 and does not understand where the problem comes from.
- 5 forms uses a URLField, which means that we would have to copy times the same validation code.

Let me know what you prefer.